docs/administering/for-work.md

```diff
@@ -0,0 +1,112 @@
+
+**Global Audit Logging.** Monitor data across the whole organization to keep track of who has
+accessed what.
+
+If you're interested in these features, we'd love to hear from you. Please [contact
+us](https://sandstorm.io/business)!
+
+## How to get Sandstorm for Work
+
+### Installing Sandstorm for Work
+
+Sandstorm comes bundled with the code needed to convert it into a Sandstorm for Work installation.
+
+If you don't have Sandstorm yet, follow these steps first.
+
+- [Install Sandstorm](https://sandstorm.io/install) via the usual directions.
+
+- Obtain a feature key. At the moment, we send all feature keys by email to you. To request a key,
+  visit the [Sandstorm for Work page on our website](https://sandstorm.io/business), click **Contact
+  Us**, fill out the form, and mention that you want to unlock Sandstorm for Work.
+
+Now that you have Sandstorm, enable Sandstorm for Work. These steps work properly on any up-to-date
+Sandstorm install, even if you installed before Sandstorm for Work was avaiable.
+
+- Log in as an administrator.
+
+- Click on your name, then click on **Admin Settings** to visit `/admin/settings`.
+
+- Click on **For Work** to visit `/admin/features`.
+
+- Click **Choose File** to upload a feature key file. You can alternatively paste a feature key into
+  the text area and click Verify.
+
+Once you've done this, you will see information about your feature key on the **For Work** tab. All
+Sandstorm features enabled by your feature key will be available to you in the most
+contextually-relevant location.
+
+### All about feature keys
+
+A feature key is a text file of this format:
+
+```
+--------------------- BEGIN SANDSTORM FEATURE KEY ----------------------
+AAAAB3NzaC1yc2EAAAADAQABAAABAQC80vEoj2Mgpprswcj5WmWY4KLwU/SWb6UE+FVpHg6+
+qwVpCggjJiPYH/WZX7d4tuqXtifx6uuQp1Pm8So86ke3AQODHFmAVgqt19QcWu1LkEFEL1c2
+4RhL8gM8lxpzWBn/3eRZ+rdUNSaVJwrXHRetjetwksfyaByQwApSphip2+HGSMxlEqATg5uh
+mxR0PzpfIwLxun8rc18j8JZQLHUim1njS8X/p7E3s9/6HeGz
+---------------------- END SANDSTORM FEATURE KEY -----------------------
+```
+
+It is a text serialization of a [Cap'n Proto](https://capnproto.org/) structure, defined in
+[feature-key.capnp](https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/feature-key.capnp).
+Feature keys are signed by an ed25519 signing key.
+
+If you are developing Sandstorm, and you need to enable Sandstorm for Work features, you can get a
+testing feature key by contacting Kenton Varda. You must also can enable testing mode by setting
+`IS_TESTING=yes` in `sandstorm.conf` for testing keys to be considered valid. Note that enabling
+testing mode forfeits all security.
+
+### Special pricing
+
+We're grateful for all the work of non-profits and community groups. For that reason, non-profits,
+charities, and small unincorporated community groups can often receive feature keys for free. Please
+[contact us via the Sandstorm for Work page](https://sandstorm.io/business) and tell us one to two
```

Comment:
Does the form have a checkbox to indicate if someone is a non-profit? If you want them to contact you through the form, it should, so you know how to address them.